consensus/src/test/java/org/apache/iotdb/consensus/iot/StabilityTest.java

```diff
@@ -96,9 +103,10 @@ public void snapshotTest() throws Exception {
             Collections.singletonList(new Peer(dataRegionId, 1, new TEndPoint("0.0.0.0", 9000))));
```

Comment:
You should call `deletePeer` here to eliminate the relationship between different tests

Reply:
Aleady fixed.